src/platforms/common/configuration/releases.mdx

```diff
@@ -46,26 +48,38 @@ The value can be arbitrary, but we recommend either of these naming strategies:
   - `version` is the semver-like structure `<major>.<minor?>.<patch?>.<revision?>-<prerelease?>` (`CFBundleShortVersionString` on iOS, `versionName` on Android)
   - `build` is the number that identifies an iteration of your app (`CFBundleVersion` on iOS, `versionCode` on Android)
-- **Commit SHA**: If you use a DVCS, we recommend using the identifying hash (for example, the commit SHA, `da39a3ee5e6b4b0d3255bfef95601890afd80709`). You can let Sentry CLI automatically determine this hash for supported version control systems with `sentry-cli releases propose-version`.
+- **Commit SHA**: If you use a version control system like Git, we recommend using the identifying hash (for example, the commit SHA, `da39a3ee5e6b4b0d3255bfef95601890afd80709`). You can let Sentry CLI automatically determine this hash for supported version control systems. See [Sentry CLI Documentation](/product/cli/releases/#creating-releases).
 
 <Note>
 
 Releases are global per organization; prefix them with something project-specific for easy differentiation.
 
 </Note>
 
-<PlatformContent includePath="set-release" notateUnsupported />
+## Setting a release
+
+How you make the release name (or version) available to your code is up to you. For example, you could use an environment variable that is set during the build process or during initial start-up.
 
-How you make the version available to your code is up to you. For example, you could use an environment variable that is set during the build process.
+Setting the release name tags each event with that release name. We recommend that you tell Sentry about a new release before sending events with that release name, as this will unlock a few more features. See our documentation about [releases](/product/releases/).
 
-This tags each event with the release value. We recommend that you tell Sentry about a new release before deploying it, as this will unlock a few more features as discussed in our documentation about [releases](/product/releases/). But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
+If you do not tell Sentry about a new release, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
 
 After configuring your SDK, you can install a repository integration or manually supply Sentry with your own commit metadata. Read our documentation about [setting up releases](/product/releases/setup/) for further information about integrations, associating commits, and telling Sentry when deploying releases.
 
 <PlatformSection supported={["apple", "android", "javascript", "flutter", "native", "rust", "python", "react-native", "dotnet", "unity"]}>
 
 ## Release Health
-  
+
+### Sessions
+
+A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed) and others.
```

Comment:
Ending that second sentence with "and others" leaves it open for the question "what else?". If we're not prepared to answer that, then maybe we should just end the list at status. If you're dead set on keeping that, then I've made an edit to fix the wording. 
 
```suggestion
A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed), and other information.
```